core/rfb.js

```diff
@@ -4104,6 +4183,16 @@ export default class RFB extends EventTargetMixin {
         this._updateClip();
         this._updateScale();
 
+        /*
```

Comment:
Do we want to keep commented code in? Might be worth explaining what the code is for if we aren't removing it.